Library/Homebrew/cli/parser.rb

```diff
@@ -64,10 +64,21 @@ def switch(*names, description: nil, env: nil, required_for: nil, depends_on: ni
           set_constraints(name, required_for: required_for, depends_on: depends_on)
         end
 
-        enable_switch(*names, from: :env) if !env.nil? && !ENV["HOMEBREW_#{env.to_s.upcase}"].nil?
+        enable_switch(*names, from: :env) if env?(env)
       end
       alias switch_option switch
 
+      def env?(env)
```

Comment:
Nice refactor. 👏 How about something like (if I've understood this correctly):
```ruby
def env?(env)
return false if env.blank?
return true if env == :verbose && ENV["VERBOSE"]

ENV["HOMEBREW_#{env.to_s.upcase}"].present?
```

Reply:
(although `VERBOSE` will never actually be set?)

Reply:
Yeah you've understood this correctly 👍  Updating the code.